show.go

```diff
@@ -41,7 +41,7 @@ func show(id int, format string, ssort string, reverse bool, tag string) {
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"title", "format", "tags", "rating", "trust", "url"})
+	table.SetHeader([]string{"id", "title", "format", "tags", "rating", "trust", "submitted", "updated", "url"})
```

Comment:
this is going to wrap on all but the biggest screens. I don't care about the id field it's meaningless if you have the url